install

```diff
@@ -253,13 +253,17 @@ unless mac?
   HOMEBREW_REPOSITORY = "#{HOMEBREW_PREFIX}/Homebrew".freeze
 end
 
+# Invalidate sudo timestamp before exiting (if it wasn't active before).
+at_exit { Kernel.system "/usr/bin/sudo", "-k" } if @have_sudo
+
+
```

Comment:
Please remove this chunk.

Reply:
This needs to be here because `have_sudo` is just set above.

Reply:
I believe `sudo -k` is successful whether or not the user has `sudo` access.

Reply:
Yes, both cases work in the same as a result.

I just realized that if we want to implement correctly as a comment **(if it wasn't active before)**,
we put this line after line 242.